### PR TITLE
add chart

### PR DESCRIPTION
**Pull request by TDM1911**
## Related Tickets:
- [#30688](https://edu-redmine.sun-asterisk.vn/issues/30688)

## WHAT this PR does?
- Add activities chart.

## Notes Impacted Areas:
- N/A.

## Attachments:
ảnh

### CODE REVIEW

File: app/Http/Controllers/TaskListController.php [hunk 1]
```diff
@@ -140,4 +143,58 @@ public function destroy(Project $project, TaskList $taskList)
...
+        $chart = new ActivityChart;
+        $labels = [];
+        $data = [];
+        $tomorrow = Carbon::tomorrow()->toDateString();
+        $date = Carbon::today()->subWeek();
+
+        $tasks = $taskList->tasks()
+            ->where('is_completed', 1)
+            ->orderBy('tasks.updated_at', 'desc')
+            ->select(DB::raw('count(tasks.updated_at) as activities,date(`tasks`.`updated_at`) as date'))
```
Comment (kienpham9819) [inline app/Http/Controllers/TaskListController.php]:
```suggestion
            ->select(DB::raw('count(tasks.updated_at) as activities, date(`tasks`.`updated_at`) as date'))
```

File: config/charts.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        'indigo' => '#6574cd',
+        'purple' => '#9561e2',
+        'pink' => '#f66d9b',
+        'red' => '#e3342f',
+        'orange' => '#f6993f',
+        'yellow' => '#ffed4a',
+        'green' => '#38c172',
+        'teal' => '#4dc0b5',
+        'cyan' => '#6cb2eb',
+
```
Comment (kienpham9819) [inline config/charts.php]:
```suggestion
```

**Comment by TDM1911** [comment]
ready

File: resources/views/projects/lists/show.blade.php [hunk 3]
```diff
@@ -200,6 +200,20 @@
...
+            <a class="btn btn-primary text-uppercase w-100" data-toggle="collapse" href="#chart" role="button" aria-expanded="false" aria-controls="chart">
+                {{ trans('general.chart') }}
+            </a>
+            <div class="collapse bg-white p-2 border mt-1" id="chart">
+                <div class="container p-2 chart-frame">
+                    {{ $chart->container() }}
+                    {{ $chart->script() }}
+                </div>
+                <p class="font-weight-bold mt-2 text-center text-uppercase">
+                    Activities on tasklist: {{ $taskList->name }}
```
Comment (huyenttm-0901) [inline resources/views/projects/lists/show.blade.php]: i18n

**Comment by TDM1911** [comment]
ready

**Pull request merged**